integration/image/remove_unix_test.go

```diff
@@ -63,7 +63,7 @@ func TestRemoveImageGarbageCollector(t *testing.T) {
 
 	// Build a image with multiple layers
 	dockerfile := `FROM busybox
-	RUN echo echo Running... > /run.sh`
+	RUN echo 'echo Running...'' > /run.sh`
```

Comment:
This is in `TestRemoveImageGarbageCollector`, so maybe it's me who broke things 😅 https://github.com/moby/moby/blob/773f792b8806782e6ed5b4127b844df90c3a615d/integration/image/remove_unix_test.go#L31

Reply:
Oh! double `''` - I guess my editor was trying to be helpful here 🤦